optaplanner-docs/src/main/asciidoc/Integration/Integration-chapter.adoc

```diff
@@ -479,18 +479,85 @@ This generates:
 {"bendableScore":"[0/0]hard/[-200/-20/0]soft"}
 ----
 
-
 [[integrationWithQuarkus]]
 == Quarkus
+////
+Quarkus and Spring Boot support similar configuration properties.
+We define them as attributes and use these in both sections.
+////
+:property-optaplanner-solver-manager-parallel-solver-count: The number of solvers that run in parallel. \
+This directly influences CPU consumption. \
+Defaults to `AUTO`.
+
+:property-optaplanner-solver-config-xml: A classpath resource to read the solver configuration XML. \
```

Comment:
The dots of the property
`optaplanner.solver.config-xml`
become "-", but it already contains "-", resulting in
`optaplanner-solver-config-xml`
which might mistake the property for `optaplanner.solver.config.xml` (which is wrong, the . before xml must be a -).

1) Can we use dots in adoc attributes?
I guess not or you'd have done it

2) Can we use underscores to replace the dots?

so
`:property-optaplanner_solver_config-xml:`